modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/renderer/AvailabilityLabelInfoItemRenderer.java

```diff
@@ -143,6 +143,13 @@ public void render(
 							"default"));
 				}
 			}
+			else {
+				httpServletRequest.setAttribute(
+					"liferay-commerce:availability-label:label",
+					StringPool.BLANK);
+				httpServletRequest.setAttribute(
+					"liferay-commerce:availability-label:labelType", "default");
+			}
```

Comment:
I think it's better to move this before the if and clear these values anytime there's a value coming from another request

Reply:
@andrea-ale-sbarra can you see if it's alright now? Thank you.